### PR TITLE
🌐 [i18n-KO] Translated `tasks/zero_shot_image_classification.mdx` to Korean

### PR DESCRIPTION
**Pull request by HanNayeoniee**
# What does this PR do?

Translated the `tasks/zero_shot_image_classification.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

**Comment by gabrielwithappy** [comment]
LGTM! :-)

File: docs/source/ko/tasks/zero_shot_image_classification.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+-->
+
+# 제로샷(zero-shot) 이미지 분류[[zeroshot-image-classification]]
+
+[[open-in-colab]]
+
+제로샷(zero-shot) 이미지 분류는 특정 카테고리의 예시가 포함된 데이터를 학습되지 않은 모델을 사용해 이미지 분류를 수행하는 작업입니다.
+
+일반적으로 이미지 분류를 위해서는 레이블이 달린 특정 이미지 데이터로 모델 학습이 필요하며, 이 모델은 특정 이미지의 특징을 레이블에 '매핑'하는 방법을 학습합니다.
+새로운 레이블이 있는 분류 작업에 이러한 모델을 사용해야 하는 경우에는, 모델을 '재보정'하기 위해 미세 조정이 필요합니다.
```
Comment (0525hhgus) [inline docs/source/ko/tasks/zero_shot_image_classification.mdx:20]:
```suggestion
일반적으로 이미지 분류를 위해서는 레이블이 달린 특정 이미지 데이터로 모델 학습이 필요하며, 이 모델은 특정 이미지의 특징을 레이블에 "매핑"하는 방법을 학습합니다.
새로운 레이블이 있는 분류 작업에 이러한 모델을 사용해야 하는 경우에는, 모델을 "재보정"하기 위해 미세 조정이 필요합니다.
```
원문에서 큰따옴표`"`를 사용하고 있어서, `"`로 수정했습니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다! candidate_labels과 같이 코드를 고려하여 번역해주셔서 이해가 수월했습니다 👍 
아래 짧은 리뷰 남깁니다 🙂

File: docs/source/ko/tasks/zero_shot_image_classification.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+시작하기 전에 필요한 라이브러리가 모두 설치되어 있는지 확인하세요:
+
+```bash
+pip install -q transformers
+```
+
+## 제로샷(zero-shot) 이미지 분류 파이프라인[[zeroshot-image-classification-pipeline]]
+
+[`pipeline`]을 활용하면 가장 간단하게 제로샷 이미지 분류를 지원하는 모델을 추론해볼 수 있습니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/zero_shot_image_classification.mdx:41]:
```suggestion
[`pipeline`]을 활용하면 가장 간단하게 제로샷 이미지 분류를 지원하는 모델로 추론해볼 수 있습니다.
```

File: docs/source/ko/tasks/zero_shot_image_classification.mdx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+>>> image
+```
+
+<div class="flex justify-center">
+     <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/owl.jpg" alt="Photo of an owl"/>
+</div>
+
+이미지와 해당 이미지의 후보 레이블인 candidate_labels를 파이프라인으로 전달합니다.
+여기서는 이미지를 직접 전달하지만, 컴퓨터에 저장된 이미지의 경로나 url로 전달할 수도 있습니다.
+candidate_labels는 이 예시처럼 간단한 단어일 수도 있고 좀 더 설명적인 단어일 수도 있습니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/zero_shot_image_classification.mdx:69]: 인수를 직접 명시하니 확실히 알기 쉬운 것 같아요! 
다른 문서에서 인수, 메소드, 클래스 등은 ` ` 안에 넣어서 표현하는데, 문제가 없다면 추가하는 게 어떨까요?

**Review by sim-so — CHANGES_REQUESTED** [review]
좋은 번역 덕분에 금방 읽었어요 🙌 
아래와 같이 짧은 리뷰를 덧붙입니다!

File: docs/source/ko/tasks/zero_shot_image_classification.mdx (hunk 1 above)
Comment (HanNayeoniee) [reply docs/source/ko/tasks/zero_shot_image_classification.mdx:20]: `"`로 수정했습니다!

File: docs/source/ko/tasks/zero_shot_image_classification.mdx (hunk 3 above)
Comment (HanNayeoniee) [reply docs/source/ko/tasks/zero_shot_image_classification.mdx:69]: 저도 고민하던 부분인데, 다른 문서와 동일하게 ` ` 안에 넣는게 좋겠네요!
리뷰 감사합니다 :)

**Review by sgugger — APPROVED** [review]
Thanks!

**Pull request merged**